exercise-1/index.html

```diff
@@ -6,18 +6,28 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Exercise 1</title>
     <link rel="stylesheet" href="./style.css" />
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+
   </head>
   <body>
-    <button>Menu</button>
-    <nav>
+    <div class="container">
+  <button class="hamburger">
+    <i class="menuIcon material-icons">menu</i>
+  </button>
+</button>
+    <nav class="display">
       <ul>
-        <li><a href="#">Home</a></li>
+        <li><a href="#"">Home</a></li>
```

Comment:
You have a double double quote here:
```suggestion
        <li><a href="#">Home</a></li>
```